WcaOnRails/spec/mailers/competitions_mailer_spec.rb

```diff
@@ -107,15 +107,14 @@
     let(:competitor_user) { FactoryBot.create :user, :wca_id }
     let(:mail) { CompetitionsMailer.notify_users_of_results_presence(competitor_user, competition) }
 
-    it "renders the headers" do
-      expect(mail.subject).to eq "The results of #{competition.name} are posted"
-      expect(mail.to).to eq [competitor_user.email]
-      expect(mail.from).to eq ["notifications@worldcubeassociation.org"]
-      expect(mail.reply_to).to match_array competition.all_delegates.pluck(:email)
-    end
-
-    it "renders the body" do
-      expect(mail.body.encoded).to match(/Your results at .+ have just been posted./)
+    it "renders" do
+      I18n.with_locale :fr do
```

Comment:
As it is this tests will start failing when the French translation gets updated, I think here (and for the other test) we either:
  - want to test the localized version works. In which case we need the `fr` translation to exist.
  - just want to test the email renders works in English, in which case using `I18n.with_locale :en` would be enough.

I'm definitely fine with going with the second approach!